scripts/scil_fodf_bundleparc.py

```diff
@@ -97,75 +92,68 @@ def main():
     parser = _build_arg_parser()
     args = parser.parse_args()
 
-    assert_inputs_exist(parser, [args.in_fodf, args.in_wm])
+    assert_inputs_exist(parser, [args.in_fodf])
     assert_outputs_exist(parser, args, args.out_prefix)
```

Comment:
I think here you need to generate all of your output (for loop, with all bundles and creates the expected list of output (if using out_folder, skip this, anyway the folder must be empty)